integration_test/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+PYTHON=/usr/local/bin/python
```

Comment:
That was just the one I was currently using at that moment. 
The Dockerfile is using python2.7 so I recommend that one.